reasoner/controller/ConjunctionController.java

```diff
@@ -101,7 +101,8 @@ protected void setUpUpstreamControllers() {
 
     abstract FunctionalIterator<Concludable> concludablesTriggeringRules();
 
-    List<Resolvable<?>> plan(Set<Variable.Retrievable> boundVariables) {
+    List<Resolvable<?>> plan(Set<Variable.Retrievable> bounds) {
+        Set<com.vaticle.typedb.core.pattern.variable.Variable> boundVariables = iterate(bounds).map(id -> conjunction.pattern().variable(id)).toSet();
```

Comment:
The planner deals exclusively with pattern.Variable instead of Identifier.Variable